### DISCUSSION TITLE
Add possibility to sort goals in UI

### DISCUSSION THREAD

**Discussion by divinerites** [plausible/analytics#245] [Anything Else]
## Prerequisites
- [X] I have searched open and closed issues to make sure that the feature has not yet been requested.

## Feature request

For my site I have 60 (sixty) click-goals (see #134 for reducing this number).

It would be cool to be able to sort them alphabetically (and when #134 is done, by other dimensions).

**Update:** Sorting is now possible in the Goal Conversions report by unique conversions, total conversions or the conversion rate %. You can also sort high to low or low to high.

**Comment by MilllerTime** [comment]
Changing how goals are sorted would be great!

Currently it seems like all goals in the current view are sorted by uniques/CR. I have some events where the total column is more important to me than the uniques/CR, and I'd like to be able to click on the "Total" header to sort by that column.

**Comment by metmarkosaric** [comment]
Sorting is now possible in the Goal Conversions report by unique conversions, total conversions or the conversion rate %. You can also sort high to low or low to high.
